cdap-cli/src/main/java/io/cdap/cdap/cli/commandset/DefaultCommands.java

```diff
@@ -33,7 +32,6 @@ public class DefaultCommands extends CommandSet<Command> {
   public DefaultCommands(Injector injector) {
     super(
       ImmutableList.<Command>builder()
-        .add(injector.getInstance(ExecuteQueryCommand.class))
         .build(),
```

Comment:
This is just `ImmutableList.of()` or `Collections.emptyList()`

Reply:
^^

Reply:
Yes, this was done. I forgot to resolve this.

Reply:
I don't see it done. It's still `ImmutableList.builder().build()`

Reply:
Yes, I noticed it just now. This change must have been excluded by me when I removed all the unrelated refactoring changes from this PR. I have made the final commit with this change and then I will go ahead and merge this PR.